vslib/SwitchState.cpp

```diff
@@ -199,6 +199,102 @@ void SwitchState::asyncOnLinkMsg(
     m_switchConfig->m_eventQueue->enqueue(std::make_shared<Event>(EVENT_TYPE_NET_LINK_MSG, payload));
 }
 
+sai_status_t SwitchState::getNetStat(
+        _In_ sai_stat_id_t counter_id, 
+        _In_ std::string& if_name, 
+        _Out_ uint64_t &counter)
+{
+    struct {
+        sai_stat_id_t id;
+        std::string name;
+    } counter_map [] = {
```

Comment:
this map also should be declared in SwithcState.h and ssigned in cpp file as const, no need to delcare it in function